test/hbbft_distributed_SUITE.erl

```diff
@@ -215,6 +218,207 @@ serialization_test(Config) ->
     [ unlink(W) || {_, {ok, W}} <- Workers ],
     ok.
 
+partition_test(Config) ->
```

Comment:
these two tests are highly duplicative.  the filtering is subtle (it took me a couple of reads to even find it), so maybe call that out more and explain what you're trying to provoke with it.  overall the 'why' of the tests is vague, so maybe a block comment above both explaining the strategy employed and the kinds of issues you're trying to provoke?